### PR TITLE
Added the Xamarin (Android and Forms) version of the DisplayLayerView State to be uniform with the other SDK versions.

### PR DESCRIPTION
**Pull request by dkemlage**
Added a comparable DisplayLayerViewState sample to be uniform with the other SDK versions.

### CODE REVIEW

**Comment by dkemlage** [comment]
@ThadT Could you please do the code-review on this issue?

File: src/Android/Xamarin.Android/Samples/MapView/DisplayLayerViewState/DisplayLayerViewState.cs [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+            // Event for layer view state changed
+            _myMapView.LayerViewStateChanged += OnLayerViewStateChanged;
+
+            // Provide used Map to the MapView
+            _myMapView.Map = myMap;
+        }
+
+        private void OnLayerViewStateChanged(object sender, LayerViewStateChangedEventArgs e)
+        {
+            // For each execution of the MapView.LayerViewStateChanges Event, get the name of
```
Comment (ThadT) [inline src/Android/Xamarin.Android/Samples/MapView/DisplayLayerViewState/DisplayLayerViewState.cs]: Typo: MapView.LayerViewStateChange**d** event

File: src/Android/Xamarin.Android/Samples/MapView/DisplayLayerViewState/DisplayLayerViewState.cs [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                "http://sampleserver6.arcgisonline.com/arcgis/rest/services/Recreation/FeatureServer/0");
+
+            // Create a feature layer using url
+            FeatureLayer myFeatureLayer = new FeatureLayer(featureLayerUri);
+            myFeatureLayer.Name = "Feature Layer";
+
+            // Add the feature layer to map
+            myMap.OperationalLayers.Add(myFeatureLayer);
+
+            // Create a mappoint the map should zoom to
```
Comment (ThadT) [inline src/Android/Xamarin.Android/Samples/MapView/DisplayLayerViewState/DisplayLayerViewState.cs]: Typo: should be either "MapPoint" (the object) or "map point" (generic)

File: src/Android/Xamarin.Android/Samples/MapView/DisplayLayerViewState/DisplayLayerViewState.cs [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+            _myMapView.LayerViewStateChanged += OnLayerViewStateChanged;
+
+            // Provide used Map to the MapView
+            _myMapView.Map = myMap;
+        }
+
+        private void OnLayerViewStateChanged(object sender, LayerViewStateChangedEventArgs e)
+        {
+            // For each execution of the MapView.LayerViewStateChanges Event, get the name of
+            // the layer and it's LayerViewState.Status
```
Comment (ThadT) [inline src/Android/Xamarin.Android/Samples/MapView/DisplayLayerViewState/DisplayLayerViewState.cs]: Typo: "its" not "it's"

**Comment by ThadT** [comment]
@dkemlage Just a couple of ticky-tack things (minor typos). Otherwise looks good ... ready to merge.

**Comment by dkemlage** [comment]
@ThadT  Thanks for the QA. I incorporated the changes into the code files for the Xamarin versions of the DisplayLayerViewState sample.

**Pull request merged**